v23.1/execution-locality-backups.md

```diff
@@ -0,0 +1,100 @@
+
+{{site.data.alerts.callout_info}}
+See [Take and Restore Locality-aware Backups](take-and-restore-locality-aware-backups.html) to run backup jobs where each node writes files only to the backup storage bucket that matches the node locality configured at node startup.
+{{site.data.alerts.end}}
+
+## Syntax
+
+To specify the locality requirements for the coordinating node, run `EXECUTION LOCALITY` with key-value pairs. The key-value pairs represent the [locality designations](cockroach-start.html#locality) assigned to the cluster at startup.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BACKUP DATABASE {database} INTO 'external://backup_storage' WITH EXECUTION LOCALITY = 'region={region},cloud={cloud}';
+~~~
+
+When you run `EXECUTION LOCALITY`, consider the following:
+
+- The backup job will fail if no nodes match the locality filter.
+- Selection of the coordinating node that matches the locality filter may slightly increase the startup latency of the backup job.
+- Even though a backup job has been pinned to a locality, it may still read data from another locality if there are no replicas in its locality.
+- If the job is created on a node that does not match the locality filter, you will receive an error even when the **job creation was successful**. This error indicates that the job execution moved to another node. To avoid viewing this error, we recommend using the [`DETACHED`](backup.html#detached) option with `EXECUTION LOCALITY` when creating a manual job rather than a [scheduled job](create-schedule-for-backup.html). Then, use the [`SHOW JOB WHEN COMPLETE`](show-jobs.html#show-job-when-complete) statement to determine when the job has finished. See [Job coordination using the execution locality option](#job-coordination-using-the-execution-locality-option) for more detail.
+
+## Job coordination using the execution locality option
+
+When you start or [resume](resume-job.html) a backup with `EXECUTION LOCALITY`, it is necessary to determine the [coordinating node for the job](backup-architecture.html#job-creation-phase). If a node that does not match the locality filter is the first node to claim the job, it will find a node that does match the filter and transfer the execution to it. This can result in a short delay in starting or resuming a backup job that has execution locality requirements.
+
+If you create a backup job on a [gateway node](architecture/sql-layer.html#overview) with a locality filter that does **not** meet the filter requirement in `EXECUTION LOCALITY`, the job will return an error indicating that it moved execution to another node. This error is returned because when you create a job without the [`DETACHED`](backup.html#detached) option, the [job execution](backup-architecture.html#resolution-phase) needs to be run to completion by the gateway node while it is still attached to the SQL client to return the result. 
```

Comment:
```suggestion
If you create a backup job on a [gateway node](architecture/sql-layer.html#overview) with a locality filter that does **not** meet the filter requirement in `EXECUTION LOCALITY`, and the job does not use the [`DETACHED`](backup.html#detached) option, the job will return an error indicating that it moved execution to another node. This error is returned because when you create a job without the `DETACHED` option, the [job execution](backup-architecture.html#resolution-phase) must run to completion before the SQL client is detached.
```
Am I understanding?